api-reference/_hidden/dxDataGridColumn/calculateGroupValue.md

```diff
@@ -13,9 +13,295 @@ The current row's data.
+            <DxColumn
+                data-field="HireDate"
+                data-type="date"
+                :group-index="0"
+                :calculate-group-value="groupByDate"
+            />
+        </DxDataGrid>
+    </template>
+    <script>
+    import { DxDataGrid, DxColumn } from 'devextreme-vue/data-grid';
+
+    export default {
+        components: {
+            DxDataGrid,
+            DxColumn
+        },
+        methods: {
+            groupByDate(rowData) {
+                const convertDateToString = (date) => {
+                    return date.toISOString().split("T")[0].replaceAll("-", "/");
+                };
+
+                const date = rowData.HireDate;
+                const yesterdayDate = new Date();
+                yesterdayDate.setDate(yesterdayDate.getDate() - 1);
+
+                const today = convertDateToString(new Date());
+                const yesterday = convertDateToString(yesterdayDate);
+
+                if (date === today) {
+                    return "Today";
+                }
+                if (date === yesterday) {
+                    return "Yesterday";
+                }
+                //...
+                return "Older";
+            }
+        }
+        // ...
+    }
+    </script>
+
+##### React
+
+    <!-- tab: App.js -->
+    import React from 'react';
+    import DataGrid, { Column } from 'devextreme-react/data-grid';
+
+    const groupByDate = (rowData) => {
+        const convertDateToString = (date) => {
+            return date.toISOString().split("T")[0].replaceAll("-", "/");
+        };
+
+        const date = rowData.HireDate;
+        const yesterdayDate = new Date();
+        yesterdayDate.setDate(yesterdayDate.getDate() - 1);
+
+        const today = convertDateToString(new Date());
+        const yesterday = convertDateToString(yesterdayDate);
+
+        if (date === today) {
+            return "Today";
+        }
+        if (date === yesterday) {
+            return "Yesterday";
+        }
+        //...
+        return "Older";
+    };
+
+    function App() {
+        // ...
+        return (
+            <DataGrid ...>
+                <Column
+                    dataField="HireDate"
+                    dataType="date"
+                    groupIndex={0}
+                    calculateGroupValue={groupByDate}
+                />
+            </DataGrid>
+        );
+    }
+
+    export default App;
+
+---
+
+Group captions and appearance can be customized. To do this, define the [groupCellTemplate](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/#groupCellTemplate) ([groupCellRender](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/#groupCellRender) in React). Refer to the following GitHub repository for an example: <a href="https://github.com/DevExpress-Examples/DataGrid---How-to-apply-custom-sorting-to-a-grouped-column" target="_blank">DataGrid - How to apply custom sorting to a grouped column</a>.
```

Comment:
For now, we don't make a distinction between React and other frameworks when it comes to template properties. When we do, we will update all texts at once.

```suggestion
Group captions and appearance can be customized. To do this, define the [groupCellTemplate](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/#groupCellTemplate). Refer to the following GitHub repository for an example: <a href="https://github.com/DevExpress-Examples/DataGrid---How-to-apply-custom-sorting-to-a-grouped-column" target="_blank">DataGrid - How to apply custom sorting to a grouped column</a>.
```